### ISSUE TITLE
Build failing 

### ISSUE THREAD

**Issue by mbbroberg** [devrelcollective/awesome-devrel#37]
Hey @jjasghar, the awesomebot we added via #36 is breaking without clear reason why. Is there an improvement to it you can offer up to report what's going on? 

![Screenshot from 2021-02-01 09-20-44](https://user-images.githubusercontent.com/1744971/106478559-f08ec280-646e-11eb-8246-d68e9f33cd72.png)

**Comment by jjasghar** [comment]
Checking into it.

**Comment by jjasghar** [comment]
It seems the last build: https://github.com/devrelcollective/awesome-devrel/runs/1807654878#step:4:130 was by the default settings an issue?

**Comment by RCheesley** [comment]
I think this is because a load of the links are broken or redirecting.  We will need to take a look and update the broken links :/

**Comment by davidgs** [comment]
Is this resolved? Can we address it or close it please?

**Comment by RCheesley** [comment]
It is still failing I think, I believe it was due to broken URLs in the
file. I hadn't had a chance to review it / fix the URLs yet though.

**Comment by shinyzhu** [comment]
#64 should work for fixing this :) 

But just for the action. Broken links need to be updated manually.

**Comment by shinyzhu** [comment]
Changes made:

```
Issues :-(
> Links 
  01. [L044]  http://www.dzonesoftware.com/ Failed to open TCP connection to www.dzonesoftware.com:80 (getaddrinfo: Temporary failure in name resolution) 
  02. [L072] 403 https://codepen.io/  
  03. [L084]  https://x.ai/ SSL_connect returned=1 errno=0 state=error: certificate verify failed (unable to get local issuer certificate) 
  04. [L097] 403 https://www.g2.com/categories/virtual-event-platforms  
  05. [L117]  https://swagger.io/ incorrect header check 
  06. [L128]  https://sarahah.com/ Failed to open TCP connection to sarahah.com:443 (getaddrinfo: Temporary failure in name resolution) 
  07. [L135]  http://gifbrewery.com/ Failed to open TCP connection to gifbrewery.com:80 (getaddrinfo: Name or service not known) 
  08. [L158]  https://gitpitch.com/ Failed to open TCP connection to gitpitch.com:443 (getaddrinfo: No address associated with hostname) 
  09. [L185]  https://blogs.oracle.com/javamagazine/ Net::ReadTimeout 
  10. [L196]  https://evman.io Failed to open TCP connection to evman.io:443 (Network is unreachable - connect(2) for "evman.io" port 443) 
  11. [L218] 403 https://www.bhphotovideo.com/c/product/1374480-REG/logitech_960_001087_c922_pro_stream_webcam.html  
  12. [L220] 403 https://www.bhphotovideo.com/c/product/1076400-REG/sony_ilce_5100l_b_alpha_a5100_mirrorless_digital.html  
  13. [L233] 410 https://www.ebags.com/product/travelpro/maxlite-5-21-expandable-carry-on-spinner/363078?productid=10688631  
```

Solution:

```
01.removed
02.can access from browser
03.removed
04.can access from browser, first brought cloudflare validation
05.can access from browser
06.removed
07.removed
08.removed
09.can access from browser
10.removed the evman.io link, keeping the GitHub repo link
11.can access from browser
12.can access from browser
13.removed
```

I also added more parameters to `awesome_bot` to skip some erros. Added `swagger.io` and `oracle.com` to the white list.

**Comment by jerdog** [comment]
Fixed with #64 changes
